clients/v1_repl/v1_repl.go

```diff
@@ -111,6 +266,7 @@ func (c *Client) executor(cmd string) {
 	if cmd == "" {
 		return
 	}
+	defer c.writeCommandToHistory(cmd)
```

Comment:
Yeah, I didn't think that having `quit`/`exit` in the command history would be useful for the user.
If you think it would be, I can just have it write before exiting, let me know.